src/views/Lottery/components/BuyTicketsModal/EditNumbersModal.tsx

```diff
@@ -69,9 +69,7 @@ const EditNumbersModal: React.FC<{
           id="lotteryBuyEdited"
           disabled={!allComplete || isConfirming}
           endIcon={isConfirming ? <AutoRenewIcon spin color="currentColor" /> : undefined}
-          onClick={() => {
-            onConfirm()
-          }}
+          onClick={onConfirm}
```

Comment:
For this onClick event, I think we should avoid passing `event` param to the callback.

Reply:
Done